src/Sign.Core/DataFormatSigners/AzureSignToolSigner.cs

```diff
@@ -57,6 +57,8 @@ public AzureSignToolSigner(
                 ".mst",
                 ".ocx",
                 ".ps1",
+                ".ps1xml",
```

Comment:
I tried to understand the full list of extensions that could be passed in here, to see if there are any other missing ones (e.g. other PowerShell-ish extensions). It seems like the only allow-list that AzureSignTool has is adding a SIP extension for Appx extensions. Do we have a way of knowing all the extensions that we possibly could provide here? 

Aside from testing it, how do we actually know ps1xml and psd1 are supported by the underlying native signing methods?

Reply:
In the general case, I don't know.  For PowerShell, there are [6 file extensions which support signing](https://github.com/PowerShell/PowerShell/blob/2f4f585e7fe075f5c1669397ae738c554fa18391/src/System.Management.Automation/security/SecurityManager.cs#L97-L106).  I updated this PR to support .cdxml.  I can't add support for .xaml because of https://github.com/dotnet/sign/issues/778.